de.finnik.PassVault.yml

```diff
@@ -0,0 +1,36 @@
+app-id: de.finnik.PassVault
+runtime: org.freedesktop.Platform
+runtime-version: '20.08'
+sdk: org.freedesktop.Sdk
+sdk-extensions:
+  - org.freedesktop.Sdk.Extension.openjdk11
+command: passvault
+modules:
+  - name: openjdk
+    buildsystem: simple
+    build-commands:
+      - /usr/lib/sdk/openjdk11/install.sh
+  - name: passvault
+    buildsystem: simple
+    build-commands:
+      - install -D passvault /app/bin/passvault
+      - install -D PassVault-3.0.jar /app/share/PassVault-3.0.jar
+      - install -D de.finnik.PassVault.desktop /app/share/applications/de.finnik.PassVault.desktop
+      - install -D icon.svg /app/share/icons/hicolor/scalable/apps/de.finnik.PassVault.svg
+      - install -D de.finnik.PassVault.metainfo.xml /app/share/appdata/de.finnik.PassVault.metainfo.xml
+    sources:
+      - type: file
+        path: passvault
+      - type: file
+        path: PassVault-3.0.jar 
```

Comment:
This file should be fetched from some external place, like https://github.com/de-finnik/PassVault/releases.